Compilers.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27102.0
+# Visual Studio Version 16
```

Comment:
intended?

Reply:
Thanks for catching that. My `compilers.sln` keeps getting updated recently and it's been error-prone to avoid adding it... :(